source/1_engine/rex_engine/include/rex_engine/filesystem/path_defines.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#ifndef REX_PATH_DEFINES
+  #define REX_PATH_DEFINES
+
+  //-------------------------------------------------------------------------
+  // File path separator
+  #ifndef REX_FOLDER_SEPS
+    #ifdef REX_PLATFORM_WINDOWS
+      #define REX_FOLDER_SEPS "\\/"
```

Comment:
can this be converted to global, inline constexpr string_views?

Reply:
I know they're used to init a string_view, but I don't know why we don't branch on platform there instead.
we want to avoid using defines as global constants as much as possible

Reply:
Same comment ( You want to move the global constants as well and use those instead of the defines? )